Movie/Movie/MainModule/View/MovieViewCell.swift

```diff
@@ -102,18 +98,11 @@ final class MovieViewCell: UITableViewCell {
         configureConstraints()
     }
 
-    private func setupImage(movie: Movie, networkService: NetworkServiceProtocol) {
+    private func setupImage(movie: Movie, imageService: ImageServiceProtocol) {
```

Comment:
```suggestion
    private func setupImage(movie: Movie, presenter: presenterProtocol) {
```
 передавай презентер и унего дергай метод получения картинки